src/sync/transforms/branch.ts

```diff
@@ -13,16 +14,13 @@ import _ from "lodash";
  *  - Title of the associated Pull Request
  *  - Messages from up to the last 100 commits in that branch
  */
-function mapBranch(branch, repository) {
+const mapBranch = (branch, repository, useNewGHPrUrl) => {
 	const branchKeys = issueKeyParser().parse(branch.name) || [];
 	const pullRequestKeys = issueKeyParser().parse(
 		branch.associatedPullRequests.nodes.length ? branch.associatedPullRequests.nodes[0].title : ""
 	) || [];
 	const commitKeys = issueKeyParser().parse(branch.target.message) || [];
-
-	const allKeys = branchKeys
-		.concat(pullRequestKeys)
-		.concat(commitKeys)
+	const allKeys = _.union(branchKeys, pullRequestKeys, commitKeys)
```

Comment:
Not sure what the feeling is around _ here, it was available so I used it.... can use straight JS if thats preferred?

This line is to remove duplicates.

Reply:
I don't have an opinion about it. Maybe @mboudreau can suggest?